api/v1alpha1/httpchaos_types.go

```diff
@@ -134,12 +134,12 @@ func (in *HTTPChaos) IsDeleted() bool {
 	return !in.DeletionTimestamp.IsZero()
 }
 
-// IsPaused returns whether this resource has been paused
-func (in *HTTPChaos) IsPaused() bool {
-	if in.Annotations == nil || in.Annotations[PauseAnnotationKey] != "true" {
-		return false
+// GetPause returns whether this resource has been paused
```

Comment:
If the method is renamed to `GetPause`, then this comment should be updated.

Reply:
thx! fixed.